resources/lang/en/apps.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
+    "task_list" => "Task List",
+    "current_tasks"=>"Current Tasks",
```

Comment:
Convention
Cần có 1 space trước và sau các toán tử như +, -, *, /, ., >, <, ==, {{  ...